modules/host.go

```diff
@@ -111,14 +119,33 @@ type (
 	// StorageObligation contains information about a storage obligation that
 	// the host has accepted.
 	StorageObligation struct {
+		ObligationId             types.FileContractID `json:"obligationid"`
+		FileSize                 uint64               `json:"filesize"`
+		SectorRootsCount         uint64               `json:"sectorrootscount"`
+		ContractCost             types.Currency       `json:"contractcost"`
+		LockedCollateral         types.Currency       `json:"lockedcollateral"`
+		PotentialDownloadRevenue types.Currency       `json:"potentialdownloadrevenue"`
+		PotentialStorageRevenue  types.Currency       `json:"potentialstoragerevenue"`
+		PotentialUploadRevenue   types.Currency       `json:"potentialuploadrevenue"`
+		RiskedCollateral         types.Currency       `json:"riskedcollateral"`
+		TransactionFeesAdded     types.Currency       `json:"transactionfeesadded"`
+
+		// The negotiation height specifies the block height at which the file
+		// contract was negotiated. The expiration height and the proof deadline
+		// are equal to the window start and window end. Between the expiration height
+		// and the proof deadline, the host must submit the storage proof.
 		NegotiationHeight types.BlockHeight `json:"negotiationheight"`
+		ExpirationHeight  types.BlockHeight `json:"expirationheight"`
+		ProofDeadLine     types.BlockHeight `json:"proofdeadline"`
```

Comment:
Same here, `ExpirationHeight` should appear above `NegotiationHeight` so that all the fields in the codeblock are sorted.